atd-vzd/.gitignore

```diff
@@ -1,2 +1,3 @@
 .env.staging
 .env.production
+.env.ldm-staging
```

Comment:
This snuck in here, but maybe it would be good to keep this here since some of us will be switching between LDM and non-LDM branches?